.buildkite/scripts/publish.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -euo pipefail
+
+pushDockerImage() {
+    docker build \
+        -t "${DOCKER_IMG_TAG}" \
+        --label BRANCH_NAME="${GITHUB_PR_BRANCH}" \
+        --label GIT_SHA="${BUILDKITE_COMMIT}" \
+        --label GO_VERSION="${SETUP_GOLANG_VERSION}" \
```

Comment:
This makes me wondering if that variable could have its value from `.go-version` file. So there is just one source of truth for the go version used.

Reply:
I actually was hoping to set up somehow SETUP_GOLANG_VERSION from this file, from my trying in previous RP > if I set up SETUP_GOLANG_VERSION as `cat .go-version` it is working inside steps, but not as parameter for agent, so I wanted to "play" with some hooks to see it will help